doc/makeflow.html

```diff
@@ -880,6 +886,20 @@ <h2>Table of Contents</h2>
 
 <code>makeflow --wrapper ./setupjava.sh --wrapper-input setupjava.sh example.makeflow</code>
 
+<a name=mpi><h2>Running as MPI</h2></a>
+
+<a name=makeflow-mpi><h3>Makeflow as MPI program</h3></a>
+If cctools and Makeflow has been configured and built with <tt>mpicc</tt>, then Makeflow can be ran as an MPI program by passing Makeflow and all of its arguments as arguments to <tt>mpirun</tt> or <tt>mpiexec</tt>. An example of using it as such looks like <code>mpirun -np 120 makeflow -T mpi example.makeflow</code> As the example shows, <tt>-T mpi</tt> is necessary to inform Makeflow that is is being ran as an mpi program. When doing so, Makeflow will count how many processes report back to the master/RANK0 process on each node, and use that as how many cores exist on each node. A single process on each node is then ran as the worker process, with RANK0 acting as the master process, dispatching jobs and acting as Makeflow usually does. Each worker will then share its memory fairly amongst all logical cores and the number of cores Makeflow measures that it gets to use on each node. This is done by following the equation <tt> worker_memory = (total_memory / total_logical_cores) * cores_for_worker</tt>. To overrite these options, use the commands <tt>--mpi-cores=NUM --mpi-memory=NUM</tt> to set the number of cores each worker should use, and how much memory each worker gets to use.
+
+<a name=mpi-starter><h3>Makeflow MPI Starter</h3></a>
+An alternative way of using Makeflow with MPI is to couple it with WorkQueue and submit both as an MPI job. You can do this by using <tt>makeflow_mpi_starter</tt>. This allows the user to submit a single program which will start Makeflow on the Rank0 process, and WorkQueue workers on all of the nodes. It behaves the same way as using Makeflow as an MPI program does in partitioning cores and memory on the workers. using it is very simple, as seen here:
+
+<code>
+mpiexec -np 120 makeflow_mpi_starter -m "-R 10 example.makeflow" -q "" -p 9000
```

Comment:
I get:
makeflow: No makeflow specified and file "./Makeflow" could not be found.
makeflow: Run "makeflow -h" for help with options.

Reply:
@btovar That appears to be a bug in Makeflow. When I take the makeflow string which makeflow_mpi_starter builds and try to use that in the shell, I get an error as well. I will continue to debug this.

Reply:
@btovar turns out, -R doesn't take an argument. That's my bad. Removing that from the documentation.